README.md

```diff
@@ -26,6 +26,12 @@ Dialtone Vue components can be imported directly from the package. Some componen
 import { DtInput, VALIDATION_MESSAGE_TYPES } from '@dialpad/dialtone-vue';
 ```
 
+If you are using the Vue 3 version of Dialtone, you must also import the css:
```

Comment:
Remove the `## Shadow DOM` section in the readme as it's not relevant anymore?

Reply:
yep, thanks